package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "DSC",
+  "description": "The Denver Suspension Collective",
+  "private": true,
+  "dependencies": {
+    "bootstrap": "^5.0.0",
+    "bootstrap-icons": "^1.0.0"
+  }
+}
```

Comment:
This is the only file worth looking at.

You create this, then run `npm install`.

The reset is generated.